docs/upgrade-guide.md

```diff
@@ -0,0 +1,27 @@
+# Upgrade Guide
+
+## Upgrading from @carto/react v1.0 to v1.1
+
+### executeSQL parameters via destructuring 
+
+ExecuteSQL parameters are passed using object destructuring.
```

Comment:
Shouldn't we indicate it is a 'breaking' ?

Reply:
Nope, all in in the upgrade is a breaking